src/Epicea/EpCangedSettingEvent.class.st

```diff
@@ -0,0 +1,37 @@
+Class {
+	#name : 'EpCangedSettingEvent',
```

Comment:
Shouldn't it be "Changed" instead of :Canged"?

Reply:
Updated